[2023-12-28T16:09:09.804+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-25T04:00:00+00:00/task_id=Create_Table permission to 509
[2023-12-28T16:09:09.805+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-25T04:00:00+00:00/task_id=Create_Table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-25T04:00:00+00:00/task_id=Create_Table'
[2023-12-28T16:09:09.847+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-25T04:00:00+00:00/task_id=Create_Table permission to 509
[2023-12-28T16:09:09.848+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-25T04:00:00+00:00/task_id=Create_Table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-25T04:00:00+00:00/task_id=Create_Table'
[2023-12-28T16:09:09.898+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: air_quality.Create_Table scheduled__2023-12-25T04:00:00+00:00 [queued]>
[2023-12-28T16:09:09.914+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: air_quality.Create_Table scheduled__2023-12-25T04:00:00+00:00 [queued]>
[2023-12-28T16:09:09.915+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-12-28T16:09:09.936+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): Create_Table> on 2023-12-25 04:00:00+00:00
[2023-12-28T16:09:09.942+0000] {standard_task_runner.py:57} INFO - Started process 567 to run task
[2023-12-28T16:09:09.946+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'air_quality', 'Create_Table', 'scheduled__2023-12-25T04:00:00+00:00', '--job-id', '773', '--raw', '--subdir', 'DAGS_FOLDER/airQuality.py', '--cfg-path', '/tmp/tmpbrtmp6zm']
[2023-12-28T16:09:09.948+0000] {standard_task_runner.py:85} INFO - Job 773: Subtask Create_Table
[2023-12-28T16:09:10.029+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-25T04:00:00+00:00/task_id=Create_Table permission to 509
[2023-12-28T16:09:10.031+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-25T04:00:00+00:00/task_id=Create_Table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-25T04:00:00+00:00/task_id=Create_Table'
[2023-12-28T16:09:10.035+0000] {task_command.py:416} INFO - Running <TaskInstance: air_quality.Create_Table scheduled__2023-12-25T04:00:00+00:00 [running]> on host d67dd4682ccf
[2023-12-28T16:09:10.247+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='ghifari' AIRFLOW_CTX_DAG_ID='air_quality' AIRFLOW_CTX_TASK_ID='Create_Table' AIRFLOW_CTX_EXECUTION_DATE='2023-12-25T04:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-12-25T04:00:00+00:00'
[2023-12-28T16:09:10.268+0000] {base.py:73} INFO - Using connection ID 'neonServerDB-Postgres' for task execution.
[2023-12-28T16:09:14.508+0000] {airQuality.py:139} INFO - Table air_quality Already Exist
[2023-12-28T16:09:14.566+0000] {python.py:194} INFO - Done. Returned value was: 
        CREATE TABLE air_quality (
            id serial primary key, 
            city varchar(250),
            country varchar(250), 
            unit varchar(20), 
            last_value float8, 
            lat float8,
            lng float8
        )
    
[2023-12-28T16:09:14.621+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=air_quality, task_id=Create_Table, execution_date=20231225T040000, start_date=20231228T160909, end_date=20231228T160914
[2023-12-28T16:09:14.749+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-12-28T16:09:14.802+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-12-28T16:52:18.357+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-25T04:00:00+00:00/task_id=Create_Table permission to 509
[2023-12-28T16:52:18.361+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-25T04:00:00+00:00/task_id=Create_Table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-25T04:00:00+00:00/task_id=Create_Table'
[2023-12-28T16:52:18.511+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-25T04:00:00+00:00/task_id=Create_Table permission to 509
[2023-12-28T16:52:18.513+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-25T04:00:00+00:00/task_id=Create_Table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-25T04:00:00+00:00/task_id=Create_Table'
[2023-12-28T16:52:18.667+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: air_quality.Create_Table scheduled__2023-12-25T04:00:00+00:00 [queued]>
[2023-12-28T16:52:18.703+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: air_quality.Create_Table scheduled__2023-12-25T04:00:00+00:00 [queued]>
[2023-12-28T16:52:18.706+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-12-28T16:52:18.764+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): Create_Table> on 2023-12-25 04:00:00+00:00
[2023-12-28T16:52:18.773+0000] {standard_task_runner.py:57} INFO - Started process 447 to run task
[2023-12-28T16:52:18.792+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'air_quality', 'Create_Table', 'scheduled__2023-12-25T04:00:00+00:00', '--job-id', '932', '--raw', '--subdir', 'DAGS_FOLDER/airQuality.py', '--cfg-path', '/tmp/tmpypaqt84e']
[2023-12-28T16:52:18.807+0000] {standard_task_runner.py:85} INFO - Job 932: Subtask Create_Table
[2023-12-28T16:52:19.047+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-25T04:00:00+00:00/task_id=Create_Table permission to 509
[2023-12-28T16:52:19.049+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-25T04:00:00+00:00/task_id=Create_Table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-25T04:00:00+00:00/task_id=Create_Table'
[2023-12-28T16:52:19.053+0000] {task_command.py:416} INFO - Running <TaskInstance: air_quality.Create_Table scheduled__2023-12-25T04:00:00+00:00 [running]> on host d67dd4682ccf
[2023-12-28T16:52:19.389+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='ghifari' AIRFLOW_CTX_DAG_ID='air_quality' AIRFLOW_CTX_TASK_ID='Create_Table' AIRFLOW_CTX_EXECUTION_DATE='2023-12-25T04:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-12-25T04:00:00+00:00'
[2023-12-28T16:52:19.433+0000] {base.py:73} INFO - Using connection ID 'neonServerDB-Postgres' for task execution.
[2023-12-28T16:52:19.732+0000] {airQuality.py:139} INFO - Table air_quality Already Exist
[2023-12-28T16:52:19.773+0000] {python.py:194} INFO - Done. Returned value was: 
        CREATE TABLE air_quality (
            id serial primary key, 
            city varchar(250),
            country varchar(250), 
            unit varchar(20), 
            last_value float8, 
            lat float8,
            lng float8
        )
    
[2023-12-28T16:52:19.851+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=air_quality, task_id=Create_Table, execution_date=20231225T040000, start_date=20231228T165218, end_date=20231228T165219
[2023-12-28T16:52:19.931+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-12-28T16:52:20.036+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-12-28T16:52:46.023+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-25T04:00:00+00:00/task_id=Create_Table permission to 509
[2023-12-28T16:52:46.037+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-25T04:00:00+00:00/task_id=Create_Table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-25T04:00:00+00:00/task_id=Create_Table'
[2023-12-28T16:52:46.157+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-25T04:00:00+00:00/task_id=Create_Table permission to 509
[2023-12-28T16:52:46.172+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-25T04:00:00+00:00/task_id=Create_Table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-25T04:00:00+00:00/task_id=Create_Table'
[2023-12-28T16:52:46.290+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: air_quality.Create_Table scheduled__2023-12-25T04:00:00+00:00 [queued]>
[2023-12-28T16:52:46.314+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: air_quality.Create_Table scheduled__2023-12-25T04:00:00+00:00 [queued]>
[2023-12-28T16:52:46.315+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-12-28T16:52:46.355+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): Create_Table> on 2023-12-25 04:00:00+00:00
[2023-12-28T16:52:46.375+0000] {standard_task_runner.py:57} INFO - Started process 667 to run task
[2023-12-28T16:52:46.384+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'air_quality', 'Create_Table', 'scheduled__2023-12-25T04:00:00+00:00', '--job-id', '983', '--raw', '--subdir', 'DAGS_FOLDER/airQuality.py', '--cfg-path', '/tmp/tmp9acz_y1z']
[2023-12-28T16:52:46.392+0000] {standard_task_runner.py:85} INFO - Job 983: Subtask Create_Table
[2023-12-28T16:52:46.619+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-25T04:00:00+00:00/task_id=Create_Table permission to 509
[2023-12-28T16:52:46.620+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-25T04:00:00+00:00/task_id=Create_Table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-25T04:00:00+00:00/task_id=Create_Table'
[2023-12-28T16:52:46.643+0000] {task_command.py:416} INFO - Running <TaskInstance: air_quality.Create_Table scheduled__2023-12-25T04:00:00+00:00 [running]> on host d67dd4682ccf
[2023-12-28T16:52:46.913+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='ghifari' AIRFLOW_CTX_DAG_ID='air_quality' AIRFLOW_CTX_TASK_ID='Create_Table' AIRFLOW_CTX_EXECUTION_DATE='2023-12-25T04:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-12-25T04:00:00+00:00'
[2023-12-28T16:52:46.943+0000] {base.py:73} INFO - Using connection ID 'neonServerDB-Postgres' for task execution.
[2023-12-28T16:52:51.176+0000] {airQuality.py:139} INFO - Table air_quality Already Exist
[2023-12-28T16:52:51.205+0000] {python.py:194} INFO - Done. Returned value was: 
        CREATE TABLE air_quality (
            id serial primary key, 
            city varchar(250),
            country varchar(250), 
            unit varchar(20), 
            last_value float8, 
            lat float8,
            lng float8
        )
    
[2023-12-28T16:52:51.238+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=air_quality, task_id=Create_Table, execution_date=20231225T040000, start_date=20231228T165246, end_date=20231228T165251
[2023-12-28T16:52:51.278+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-12-28T16:52:51.328+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-12-28T16:55:42.076+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-25T04:00:00+00:00/task_id=Create_Table permission to 509
[2023-12-28T16:55:42.078+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-25T04:00:00+00:00/task_id=Create_Table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-25T04:00:00+00:00/task_id=Create_Table'
[2023-12-28T16:55:42.139+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-25T04:00:00+00:00/task_id=Create_Table permission to 509
[2023-12-28T16:55:42.140+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-25T04:00:00+00:00/task_id=Create_Table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-25T04:00:00+00:00/task_id=Create_Table'
[2023-12-28T16:55:42.207+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: air_quality.Create_Table scheduled__2023-12-25T04:00:00+00:00 [queued]>
[2023-12-28T16:55:42.226+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: air_quality.Create_Table scheduled__2023-12-25T04:00:00+00:00 [queued]>
[2023-12-28T16:55:42.227+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-12-28T16:55:42.257+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): Create_Table> on 2023-12-25 04:00:00+00:00
[2023-12-28T16:55:42.265+0000] {standard_task_runner.py:57} INFO - Started process 1619 to run task
[2023-12-28T16:55:42.271+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'air_quality', 'Create_Table', 'scheduled__2023-12-25T04:00:00+00:00', '--job-id', '1202', '--raw', '--subdir', 'DAGS_FOLDER/airQuality.py', '--cfg-path', '/tmp/tmpn4abmizz']
[2023-12-28T16:55:42.273+0000] {standard_task_runner.py:85} INFO - Job 1202: Subtask Create_Table
[2023-12-28T16:55:42.362+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-25T04:00:00+00:00/task_id=Create_Table permission to 509
[2023-12-28T16:55:42.363+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-25T04:00:00+00:00/task_id=Create_Table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-25T04:00:00+00:00/task_id=Create_Table'
[2023-12-28T16:55:42.366+0000] {task_command.py:416} INFO - Running <TaskInstance: air_quality.Create_Table scheduled__2023-12-25T04:00:00+00:00 [running]> on host d67dd4682ccf
[2023-12-28T16:55:42.530+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='ghifari' AIRFLOW_CTX_DAG_ID='air_quality' AIRFLOW_CTX_TASK_ID='Create_Table' AIRFLOW_CTX_EXECUTION_DATE='2023-12-25T04:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-12-25T04:00:00+00:00'
[2023-12-28T16:55:42.549+0000] {base.py:73} INFO - Using connection ID 'neonServerDB-Postgres' for task execution.
[2023-12-28T16:55:46.740+0000] {airQuality.py:139} INFO - Table air_quality Already Exist
[2023-12-28T16:55:46.757+0000] {python.py:194} INFO - Done. Returned value was: 
        CREATE TABLE air_quality (
            id serial primary key, 
            city varchar(250),
            country varchar(250), 
            unit varchar(20), 
            last_value float8, 
            lat float8,
            lng float8
        )
    
[2023-12-28T16:55:46.790+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=air_quality, task_id=Create_Table, execution_date=20231225T040000, start_date=20231228T165542, end_date=20231228T165546
[2023-12-28T16:55:46.915+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-12-28T16:55:46.955+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
